### ISSUE TITLE
[K210] What is the current status of K210 support?I didn't make it work.

### ISSUE THREAD

**Issue by balanceTWK** [riscv/opensbi#187]
👀...

**Comment by nylon7** [comment]
Could you have shown your error message?

**Comment by hzcx998** [comment]
you can try to use rustsbi, it support k210 very well!

**Comment by z3cko** [comment]
@hzcx998 do you have experience with rustsbi and k210? Do you mind sharing?

**Comment by hzcx998** [comment]
> 

Yes, when i use rustsbi, it support mmu so good, but different with opensbi, it would not enable fpu auto, you need enable fpu by yourself.

**Comment by z3cko** [comment]
@hzcx998 can you explain more in depth? Do you have any examples or details? I understand that rustsbi helps in running MMU linux (can you confirm?)

**Comment by disdi** [comment]
can opensbi support nommu linux on [risv32 qemu target  ](https://github.com/buildroot/buildroot/blob/master/configs/qemu_riscv32_nommu_virt_defconfig) ?

**Comment by avpatel** [comment]
NoMMU Linux runs in M-mode so I don't think OpenSBI can be useful in this case.

We will be soon dropping the kendryte/k210 support from OpenSBI because Linux NoMMU doesn't need it.

Regards,
Anup

**Comment by SiFiveHolland** [comment]
NOMMU Linux runs in S-mode as of [f862bbf4cdca](https://git.kernel.org/torvalds/c/f862bbf4cdca696ef3073c5cf3d340b778a3e42a) (v6.10).

**Comment by avpatel** [comment]
Latest Linux now supports S-mode NoMMU but I still don't see any users of kendryte/k210 OpenSBI platform.
